apis/v1alpha1/opentelemetrycollector_webhook.go

```diff
@@ -62,6 +62,12 @@ func (r *OpenTelemetryCollector) Default() {
 		one := int32(1)
 		r.Spec.Replicas = &one
 	}
+
+	// if autoscaling is enabled then set default targetCPUUtilization
+	if r.Spec.MaxReplicas != nil && r.Spec.TargetCPUUtilization == nil {
```

Comment:
is this guard needed. Perhaps we could default always.

Reply:
Hmm based on your comment below I think I included the guard bc I was worried the default might override a user set value (and I also think it makes it clear this setting is only used when autoscaling is enabled). Removed the guard here and the ones in your comment below.

Reply:
Seems this guard and the next comment's are needed otherwise the e2e tests begin to fail